src/index.css

```diff
@@ -0,0 +1,69 @@
+:root {
+  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+  line-height: 1.5;
+  font-weight: 400;
+
+  color-scheme: light dark;
+  color: rgba(255, 255, 255, 0.87);
+  background-color: #242424;
+
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-text-size-adjust: 100%;
+}
```

Comment:
Where are these variables  --primary-color-1/2 , secondary-color-2, etc...  defined ? 
Why theire are not here?

Reply:
variables are located in variables.scss files is styles folder.
Is it correct to have 3 basic files css or scss:
- styles.css
- reset.css
- variables.css  ?